external/anomaly/tests/test_ote_anomaly_segmentation_training.py

```diff
@@ -0,0 +1,420 @@
+from ote_sdk.entities.label_schema import LabelSchemaEntity
+from ote_sdk.entities.subset import Subset
+from ote_sdk.entities.train_parameters import TrainParameters
+
+from ote_anomalib.data.mvtec import OteMvtecDataset
+
+from ote_sdk.configuration.helper import create as ote_sdk_configuration_helper_create
+from ote_sdk.test_suite.training_test_case import (OTETestCaseInterface,
+                                                   generate_ote_integration_test_case_class)
+from ote_sdk.test_suite.e2e_test_system import DataCollector, e2e_pytest_performance
+from ote_sdk.test_suite.training_tests_common import (make_path_be_abs,
+                                                      make_paths_be_abs,
+                                                      performance_to_score_name_value,
+                                                      KEEP_CONFIG_FIELD_VALUE,
+                                                      REALLIFE_USECASE_CONSTANT,
+                                                      ROOT_PATH_KEY)
+from ote_sdk.test_suite.training_tests_helper import (OTETestHelper,
+                                                      DefaultOTETestCreationParametersInterface,
+                                                      OTETrainingTestInterface)
+from ote_sdk.test_suite.training_tests_actions import (create_environment_and_task,
+                                                       OTETestTrainingAction,
+                                                       BaseOTETestAction,
+                                                       OTETestTrainingEvaluationAction,
+                                                       OTETestExportAction,
+                                                       OTETestExportEvaluationAction,
+                                                       OTETestPotAction,
+                                                       OTETestPotEvaluationAction,
+                                                       OTETestNNCFAction,
+                                                       OTETestNNCFEvaluationAction,
+                                                       OTETestNNCFExportAction,
+                                                       OTETestNNCFExportEvaluationAction,
+                                                       OTETestNNCFGraphAction)
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture
+def ote_test_domain_fx():
+    return 'anomaly-segmentation'
+
+@pytest.fixture(scope='session')
+def ote_templates_root_dir_fx():
+    import os.path as osp
+    import logging
+    logger = logging.getLogger(__name__)
+    root = osp.dirname(osp.dirname(osp.realpath(__file__)))
+    root = f'{root}/anomaly_segmentation/configs/'
+    logger.debug(f'overloaded ote_templates_root_dir_fx: return {root}')
+    return root
+
+def DATASET_PARAMETERS_FIELDS() -> List[str]:
+    return deepcopy(['dataset_path',
+                     ])
```

Comment:
any more params here? If not, we could maybe fit into a single line such as 

`return deepcopy(['dataset_path'])`

Reply:
corrected